README.md

```diff
@@ -9,3 +9,35 @@ Group 11 project for SEG 2900 at uOttawa
 - [Peter Bou-Farah](https://github.com/peter-bf) 300295193
 - [Kyle Labelle](https://github.com/kyle-labelle) 300295594
 - [Nimet Kurnaz](https://github.com/Nimetkurnaz) 300295474
+
+# Codes Comp
+- [Description](#description)
+- [How To Use](#how-to-use)
+- [License](#license)
+- [Authors Info](#author-info)
+
+---
+## Description
+Codes Comp is a website that measures users' coding skills as well as contributes to their coding skills. 
+Players improve their coding skills and develop algorithms by competing with their opponents. Thanks to the Code Comp website, instructions are given to the person's computer to indicate how to solve a particular problem. Therefore, while coding for the given problem, it is necessary to think about how to follow a path to solve it and what steps to take first. If an incorrect command is entered into the computer during coding, the desired result will not be achieved, and different methods will have to be tried to solve the problem. When solving problems in code snippets used, there is a need to try different approaches, so after a while, experience in problem-solving will be gained and reflections on real life. During the development of the given code, when it sees an illogical or inappropriate code snippet for the desired syntax, it will immediately give an error. This situation causes the need for alternative thinking to find the source of the error and multiple analyses from different approaches. During these operations, a critical approach to events and the further development of this skill are provided. The problem-solving approach found by the user himself will also increase the person's self-confidence when it gives the same result as other solutions. It is encouraged to try different solutions to reach the result, approach events from different perspectives, and increase creativity.
```

Comment:
This line is very long so it would be good to split up a bit. I also think that it may be more pertinent to also talk about the competitive element of the website and perhaps describe this part a bit more briefly.